test/integration/graql/query/GraqlDeleteIT.java

```diff
@@ -110,527 +110,8 @@ public static void closeSession() {
-        EntityType person = tx.putEntityType("person").plays(author);
-        EntityType production = tx.putEntityType("production").plays(work);
-        AttributeType<String> provenance = tx.putAttributeType("provenance", AttributeType.ValueType.STRING);
-        authoredBy.has(provenance);
-
-        Entity aPerson = person.create();
-        ConceptId personId = aPerson.id();
-        Relation aRelation = authoredBy.create();
-        aRelation.assign(author, aPerson);
-        Attribute<String> aProvenance = provenance.create("hello");
-        aRelation.has(aProvenance);
-        ConceptId relationId = aRelation.id();
-
-        tx.commit();
-        tx = session.transaction(Transaction.Type.WRITE);
-
-        aProvenance = tx.getAttributesByValue("hello").iterator().next();
-        assertTrue(aProvenance.type().label().toString().equals("provenance"));
-        assertEquals(aProvenance.owners().count(), 1);
-
-        aRelation = tx.getConcept(relationId);
-        tx.getConcept(personId).delete();
-        assertTrue(aRelation.isDeleted());
-
-        assertEquals(aProvenance.owners().collect(Collectors.toList()).size(), 0);
-    }
-
-
-    @Test
-    public void afterDeletingAllInstances_TheTypeCanBeUndefined() {
-        MatchClause movie = Graql.match(x.isa("movie"));
-
-        assertNotNull(tx.getEntityType("movie"));
-        assertExists(tx, movie);
-
-        tx.execute(movie.delete(x.isa("movie")));
-
-        assertNotNull(tx.getEntityType("movie"));
-        assertNotExists(tx, movie);
-
-        tx.execute(Graql.undefine(type("movie").sub("production")));
-
-        assertNull(tx.getEntityType("movie"));
-    }
-
-    @Test
-    public void whenDeletingMultipleVariables_AllVariablesGetDeleted() {
-        tx.execute(Graql.define(type("fake-type").sub(ENTITY)));
-        tx.execute(Graql.insert(x.isa("fake-type"), y.isa("fake-type")));
-
-        assertEquals(2, tx.stream(Graql.match(x.isa("fake-type"))).count());
-
-        tx.execute(Graql.match(x.isa("fake-type"), y.isa("fake-type"), x.not(y.var())).delete(x.isa("fake-type"), y.isa("fake-type")));
-
-        assertNotExists(tx, var().isa("fake-type"));
-    }
-
-    @Test
-    public void whenDeletingWithNoArguments_AllVariablesGetDeleted() {
-        tx.execute(Graql.define(type("fake-type").sub(Graql.Token.Type.ENTITY)));
-        tx.execute(Graql.insert(x.isa("fake-type"), y.isa("fake-type")));
-
-        assertEquals(2, tx.stream(Graql.match(x.isa("fake-type"))).count());
-
-        tx.execute(Graql.match(x.isa("fake-type"), y.isa("fake-type"), x.not(y.var()))
-                .delete(var("x").isa("fake-type"), var("y").isa("fake-type")));
-
-        assertNotExists(tx, var().isa("fake-type"));
-    }
-
-    @Test
-    public void whenDeletingAVariableNotInTheQuery_Throw() {
-        exception.expect(GraqlException.class);
-        exception.expectMessage(UNBOUND_DELETE_VARIABLE.getMessage(y.var()));
```

Comment:
is this documented as an error message to test in the future? Would be good to write down:

"* Test Unbound Delete Variable when variable in delete clause is not in match"

or something like this in your collection of messages to bring back tests for

Reply:
Yes, in https://github.com/graknlabs/grakn/issues/5750:

2. `match $x isa movie; delete $y isa thing;` throws "the delete clause variable [%s] is not defined in the match clause"